module/spl/spl-rwlock.c

```diff
@@ -35,11 +35,15 @@
 #if defined(CONFIG_PREEMPT_RT_FULL)
 
 #include <linux/rtmutex.h>
+#define	RT_MUTEX_OWNER_MASKALL	1UL
 
 static int
 __rwsem_tryupgrade(struct rw_semaphore *rwsem)
 {
-	ASSERT(rt_mutex_owner(&rwsem->lock) == current);
```

Comment:
It'd be nice to do this with a private `__rt_mutex_owner` wrapper macro for better readability.